### ISSUE TITLE
Gimp plugin error

### ISSUE THREAD

**Issue by jazzon** [dtschump/gmic-community#16]
Received this error when running a python plugin I had written.  The plugin works (minus the three '128') with the previous version, but fails in the current stable version (1.6.5.1) 

The plugin (the one I wrote) performs no file actions.

The three '128' values were added to correspond to the new versions color code request, but the script is other wise unchanged.

A strip down version (suitable for bug testing) is available if needed, but essentially all that is needed is the command string on the first line of this output (which shows the error).

[gmic_gimp]./error/ When running command '-gimp_plasma  1.0,7.66,2,1,0, 128, 128, 128', this error occured :
*** Error in ./gimp_plasma/*substitute/ *** Unknown filename '6-8'.
Traceback (most recent call last):
  File "/usr/lib/gimp/2.0/python/gimpfu.py", line 735, in response
    dialog.res = run_script(params)
  File "/usr/lib/gimp/2.0/python/gimpfu.py", line 361, in run_script
    return apply(function, params)
  File "/home/[snip]/.gimp-2.8/plug-ins/horiz_seamless_noise_layer.py", line 12, in do_it
    pdb.plug_in_gmic( inImage, noiseLayer, 1, gmicStr)
RuntimeError: calling error
[gmic]-1./ Start G'MIC interpreter.
[gmic]-1./gimp_plasma/*if/ Force image [0] to be in RGB mode.
[gmic]-1./gimp_plasma/*if/ Fill image [0] with random values, in range [0,255].
[gmic]-1./gimp_plasma/ Draw plasma fractal on image [0], with alpha 1, beta 7.66 and scale 2.
[gmic]-1./gimp_plasma/ Normalize image [0] in range [0,255].
[gmic]-1./gimp_plasma/*substitute/ Input file '6-8' at position 1
[gmic]-1./gimp_plasma/*substitute/ *** Error *** Unknown filename '6-8'.
[gmic]-1./gimp_plasma/ Abort G'MIC interpreter.

**Comment by dtschump** [comment]
Apparently, you have put spaces in your command:

    -gimp_plasma 1.0,7.66,2,1,0, 128, 128, 128

Try without spaces. Space is an item separator in G'MIC, so it has a very strong meaning. You cannot add spaces as you want without effect.
Try this:

    -gimp_plasma 1.0,7.66,2,1,0,128,128,128

Let me know.

**Comment by jazzon** [comment]
David,

I had already noticed your response on gimpchat before I saw this.  Thank
you for the information.

I apologize for the false report, but the plugin I was using to access that
script had previously been working correctly, and I failed to notice the
api change in the GMIC script I was calling.  The code I pasted to you was
AFTER I had already recieved errors due to API change, so I assumed it was
unreleated.

As a side note, a message informing of an invalid file action does not tell
a script writer (like me) that there is a parameter error, I was totally
confused by that ;-)

Great product/invention/project!  Keep up the great work!

Jazzon (in the gimpchat forums)

On Mon, Aug 24, 2015 at 2:12 PM, David Tschumperlé <notifications@github.com
